qutebrowser/browser/commands.py

```diff
@@ -1177,7 +1177,8 @@ def tab_move(self, index: typing.Union[str, int] = None, count=None):
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
                        maxsplit=0, no_replace_variables=True)
-    def spawn(self, cmdline, userscript=False, verbose=False, detach=False):
+    def spawn(self, cmdline, userscript=False, verbose=False,
+            output=False, detach=False):
```

Comment:
Please indent this by two more spaces to line up after the opening `(` (see e.g. the `@cmdutils.register` call above).